features/cli/oc_set_deployment_hook.feature

```diff
@@ -13,80 +13,83 @@ Feature: set deployment-hook/build-hook with CLI
   @heterogeneous @arm64 @amd64
   Scenario: OCP-11805:Workloads Set pre/mid/post deployment hooks on deployment config via oc set deployment-hook
     Given I have a project
-    When I run the :new_app client command with:
-      | template | rails-postgresql-example |
+    When I run the :create_deploymentconfig client command with:
```

Comment:
Deployment is preferred than deploymentconfig since 4.5 (or 4.6). Perhaps we should use deployment here.

Reply:
@liangxia deploy does not support pre/mid/post hooks, this feature is only for DC

Reply:
Ack